wolfssl/test.h

```diff
@@ -355,11 +355,29 @@ void join_thread(THREAD_TYPE);
 #endif
 static const word16      wolfSSLPort = 11111;
 
-static INLINE void err_sys(const char* msg)
+
+#if defined(__GNUC__)
+    #define WC_NORETURN __attribute__((noreturn))
+#else
+    #define WC_NORETURN
+#endif
+
```

Comment:
Thanks for explaining your thoughts behind it. Makes sense and I agree. We don't want to encourage WC_NORETURN use elsewhere let's keep it here.